site/reference/live/apply/README.md

```diff
@@ -1,298 +1,36 @@
-continue to support the current set of conditions.
-
-CRDs should also set the `observedGeneration` field in the status object, a
-pattern already common in the built-in types. The controller should update
-this field every time it sees a new generation of the resource. This allows
-the kpt library to distinguish between resources that do not have any
-conditions set because they are fully reconciled, from resources that have no
-conditions set because they have just been created.
-
-An example of a resource where the latest change has been observed by
-the controller which is currently in the process of reconciling would be:
-
-```yaml
-apiVersion: example.com
-kind: Foo
-metadata:
-  generation: 12
-  name: bar
-spec:
-  replicas: 1
-status:
-  observedGeneration: 12
-  conditions:
-  - lastTransitionTime: "2020-03-25T21:20:38Z"
-    lastUpdateTime: "2020-03-25T21:20:38Z"
-    message: Resource is reconciling
-    reason: Reconciling
-    status: "True"
-    type: Reconciling
-  - lastTransitionTime: "2020-03-25T21:20:27Z"
-    lastUpdateTime: "2020-03-25T21:20:39Z"
-    status: "False"
-    type: Stalled
-```
-
-The status for this resource state will be InProgress. So if the
-`--reconcile-timeout` flag is set, kpt live apply will wait until
-the `Reconciling` condition is `False` before pruning and exiting.
-
-### Examples
-
-{{% hide %}}
-
-<!-- @makeWorkplace @verifyExamples-->
-```
-# Set up workspace for the test.
-TEST_HOME=$(mktemp -d)
-cd $TEST_HOME
-```
-
-<!-- @fetchPackage @verifyExamples-->
-```shell
-export SRC_REPO=https://github.com/GoogleContainerTools/kpt.git
-kpt pkg get $SRC_REPO/package-examples/helloworld-set@next my-dir
-```
-
-<!-- @createKindCluster @verifyExamples-->
-```
-kind delete cluster && kind create cluster
-```
-
-<!-- @installResourceGroup @verifyExamples-->
-```
-kpt live install-resource-group
-```
-
-<!-- @initCluster @verifyExamples-->
-```
-kpt live init my-dir
-```
-
-{{% /hide %}}
-
-<!--mdtogo:Examples-->
-<!-- @liveApply @verifyExamples-->
-```shell
-# apply resources and prune
-kpt live apply my-dir/
-```
-
-<!-- @liveApply @verifyExamples-->
-```shell
-# apply resources and wait for all the resources to be reconciled before pruning
-kpt live apply --reconcile-timeout=15m my-dir/
-```
-
-<!-- @liveApply @verifyExamples-->
-```shell
-# apply resources and specify how often to poll the cluster for resource status
-kpt live apply --reconcile-timeout=15m --poll-period=5s my-dir/
-```
-<!--mdtogo-->
-
 ### Synopsis
 <!--mdtogo:Long-->
 ```
-kpt live apply DIR [flags]
+kpt live apply [DIR|-] [flags]
 ```
 
 #### Args
-
 ```
-DIR:
-  Path to a package directory.  The directory must contain exactly
-  one ConfigMap with the inventory object annotation.
+DIR|-:
+  Path to a directory containing KRM resources and a Kptfile with inventory
+  information. The path can be relative or absolute. Providing '-' instead
+  of the path to a directory cause kpt to read from stdin.
 ```
 
 #### Flags
```

Comment:
should the ex-global flags (your other PR) be listed here?

Reply:
Those apply to all commands in `live` command group, so I think those should be included in the `live` page. Similar, we probably want to document the global flags on the top-level group. 
Alternatively we can look into ways to include subtemplates so we can get the complete set of flags for every command, but this will lead to a pretty long list of flags.

Reply:
If this looks good otherwise, I can do it as a follow-up.